Forward `cval`  in _generic_edge_filter

michaelbratsch opened the pull request:
## Description

The `cval` argument is not used in https://github.com/scikit-image/scikit-image/blob/be7ff3442864f8c44236c4cd50c04039a85b3ea8/skimage/filters/edges.py#L126

It has been forgotten to be forwarded to scipy ndimage convolve (https://docs.scipy.org/doc/scipy/reference/generated/scipy.ndimage.convolve.html).

Relates: https://github.com/scikit-image/scikit-image/issues/7825

## Release note

```release-note
Forward `cval` in `_generic_edge_filter`
```

hmaarrfk commented:
While the CIs are failing for an unrelated reason, we typically encourage contributors to add a test so that their fix doesn't get "broken" in the near future.

michaelbratsch commented:
> While the CIs are failing for an unrelated reason, we typically encourage contributors to add a test so that their fix doesn't get "broken" in the near future.

I know I should have added one right away. Now finally found the time to add one.

michaelbratsch commented:
> I updated the release note, to be a bit more useful to users reading the release notes

Agree that this is more useful for the users. I added the other filters that use the `_generic_edge_filter`, too.
